translator/translate/otel/pipeline/applicationsignals/translator_test.go

```diff
@@ -207,6 +229,28 @@ func TestTranslatorMetricsForEC2(t *testing.T) {
 			detector:   eksdetector.TestEKSDetector,
 			isEKSCache: eksdetector.TestIsEKSCacheEKS,
 		},
+		"WithAppSignalsAndLoggingEnabled": {
```

Comment:
I think we are missing translation tests that actually test against yaml. Maybe we can extend this config? https://github.com/aws/amazon-cloudwatch-agent/blob/main/translator/tocwconfig/sampleConfig/base_appsignals_config.yaml